gateway/src/kaazing/gateway/internal/wsn/WebSocketNativeDelegateHandler.js

```diff
@@ -84,10 +81,7 @@ var WebSocketNativeDelegateHandler = (function () {
             channel._delegate.setIdleTimeout(timeout);
         } catch (e) {
             // ignore exceptions thrown by older WebSocket impls in browsers
-            ;
```

Comment:
That was due to my recent code formatting. I used IntelliJ code formatting tool to fix the indentation issue which caused this issue. They should all be fixed now.